vision/mnist/conv.jl

```diff
@@ -20,12 +20,14 @@ tX = cat(float.(MNIST.images(:test)[1:1000])..., dims = 4) |> gpu
 tY = onehotbatch(MNIST.labels(:test)[1:1000], 0:9) |> gpu
 
 m = Chain(
-  Conv((2,2), 1=>16, relu),
-  x -> maxpool(x, (2,2)),
-  Conv((2,2), 16=>8, relu),
-  x -> maxpool(x, (2,2)),
-  x -> reshape(x, :, size(x, 4)),
-  Dense(288, 10), softmax) |> gpu
+    Conv((3, 3), 1=>32, relu),
+    Conv((3, 3), 32=>32, relu),
```

Comment:
Equivalent to running a 5x5 kernel. 3x3 kernels are normally optimized better in some GPUs.